app/controllers/modules/case_logs_filter.rb

```diff
@@ -1,23 +1,16 @@
 module Modules::CaseLogsFilter
-  def filtered_case_logs(logs)
-    if session[:case_logs_filters].present?
-      filters = JSON.parse(session[:case_logs_filters])
-      filters.each do |category, values|
-        next if Array(values).reject(&:empty?).blank?
-        next if category == "organisation" && params["organisation_select"] == "all"
-
-        logs = logs.public_send("filter_by_#{category}", values, current_user)
-      end
-    end
-    logs = logs.order(created_at: :desc)
-    current_user.support? ? logs.all.includes(:owning_organisation, :managing_organisation) : logs
+  def filtered_case_logs(logs, search_term, filters)
```

Comment:
Why is the filtering logic changing? Shouldn't we be returning the same records in the CSV as before?

Reply:
The logic isn't (shouldn't be!) changing, but needed commonising between the job and this screen.